Assets/MixedRealityToolkit/_Core/Interfaces/Devices/IMixedRealityController.cs

```diff
@@ -59,5 +60,10 @@ public interface IMixedRealityController
         /// Mapping definition for this controller, linking the Physical inputs to logical Input System Actions
         /// </summary>
         MixedRealityInteractionMapping[] Interactions { get; }
+
+        /// <summary>
+        /// The transform of this controller, used for tracking and observing the controller if it's tracked.
+        /// </summary>
+        Transform Transform { get; }
```

Comment:
The device layer shouldn't know about anything in the scene.

Reply:
Sure, I'll open a PR

Reply:
>The device layer shouldn't know about anything in the scene.

Can you point me to the requirement in the approved architecture that states this?

Reply:
see https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2784#issuecomment-421424132